wscript

```diff
@@ -61,6 +61,8 @@ cxx_compiler['linux'] = ['clang++', 'g++']
 
 def options(opt):
   opt.load('compiler_cxx')
+  opt.add_option('--use-system-clang', dest='use_system_clang', default=False,
+                 action='store_true', help='use clang from the system instead of downloading it')
```

Comment:
I would note here that the recommended way to build cquery is still to use the downloaded Clang, not system Clang.

Reply:
+1

Reply:
I'll add a clarification: "(not recommended)"